evalml/data_checks/highly_null_data_check.py

```diff
@@ -9,45 +9,62 @@
 
 
 class HighlyNullDataCheck(DataCheck):
-    """Checks if there are any highly-null columns in the input."""
+    """Checks if there are any highly-null columns and rows in the input."""
 
     def __init__(self, pct_null_threshold=0.95):
-        """Checks if there are any highly-null columns in the input.
+        """Checks if there are any highly-null columns and rows in the input.
 
         Arguments:
             pct_null_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
-                that feature will be considered highly-null. Defaults to 0.95.
+                that column/row will be considered highly-null. Defaults to 0.95.
```

Comment:
Ah, I do think it would be helpful to have a separate threshold for rows vs columns.

I filed #2270 to track that separately.